.github/workflows/python-package.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
There are more elegant ways to lint python code. We use [pre-commit](https://pre-commit.com/). Linters can be configured in a `.pre-commit-config.yaml` in the repository root. If you want to keep linting in the Github Action there are actions for that, too: https://github.com/marketplace/actions/lint-action